### ISSUE TITLE
Condition Issue if it has :

### ISSUE THREAD

**Issue by KhrysAK47** [Ajneb97/ConditionalEvents#35]
Hello, I am currently using %checkitem_amount_namecontains:Leviathan Scale Talisman%
to check an item from inventory, and it looks like it is not recognized properly because of the : in it.

**Comment by KhrysAK47** [comment]
> Why do you think is not being recognized? Did you used /ce debug to check if the variable is being replaced?

Correct, /ce debug and /ce verify shows it as incorrect placeholder, but it is a correct placeholder, even when parsing through just /papi parse. Which is why I assume it has something to do with the : in the placeholder itself.

**Comment by KhrysAK47** [comment]
`'%checkitem_amount_namecontains:Leviathan Scale Talisman% >= 1'`

**Comment by Ajneb97** [comment]
The /ce verify telling you it is incorrect is a known issue with placeholders with empty spaces, but it just a warning message. Send me a picture of the /ce debug command when the event gets executed
